Source/WebCore/platform/TextRecognitionResult.h

```diff
@@ -80,6 +80,12 @@ struct TextRecognitionDataDetector {
     {
     }
 
+    TextRecognitionDataDetector(RetainPtr<DDScannerResult>&& scannerResult, Vector<FloatQuad>&& quads)
```

Comment:
Instead of doing this, let's just make the existing constructor take a RetainPtr.